packages/core-components/src/layout/Sidebar/MobileSidebar.tsx

```diff
@@ -0,0 +1,177 @@
+/*
+ * Copyright 2020 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { BackstageTheme } from '@backstage/theme';
+import {
+  BottomNavigation,
+  Box,
+  IconButton,
+  makeStyles,
+  Typography,
+} from '@material-ui/core';
+import CloseIcon from '@material-ui/icons/Close';
+import MenuIcon from '@material-ui/icons/Menu';
+import React, { createContext, useEffect, useState } from 'react';
+import { useLocation } from 'react-router';
+import { sidebarConfig } from './config';
+import { SidebarGroup } from './SidebarGroup';
+
+type MobileSidebarContextType = {
+  selectedMenuItemIndex: number;
+  setSelectedMenuItemIndex: React.Dispatch<React.SetStateAction<number>>;
+};
+
+const useStyles = makeStyles<BackstageTheme>(theme => ({
+  root: {
+    position: 'fixed',
```

Comment:
Regarding the `100vh` issue, perhaps we can just modify the flex flow in the sidebar page to lay out the mobile sidebar beneath the content page?

Reply:
Yeah - this could be smth. interesting to generally solve the Sidebar placement differently. Having it `position: fixed` & then just adding padding is probably not the best solution (for mobile as well as desktop)